rom/dev/src/flow/cold_reset/ldev_id.rs

```diff
@@ -62,23 +66,30 @@ impl LocalDevIdLayer {
         Self::derive_cdi(env, KEY_ID_FE, KEY_ID_ROM_FMC_CDI)?;
 
         // Derive DICE Key Pair from CDI
-        let key_pair = Self::derive_key_pair(env, KEY_ID_ROM_FMC_CDI, KEY_ID_LDEVID_PRIV_KEY)?;
+        let key_pair =
+            Self::derive_key_pair(env, KEY_ID_ROM_FMC_CDI, KEY_ID_LDEVID_ECDSA_PRIV_KEY)?;
 
         // Generate the Subject Serial Number and Subject Key Identifier.
         //
         // This information will be used by the next DICE Layer while generating
         // certificates
-        let subj_sn = X509::subj_sn(env, &key_pair.pub_key)?;
+        let ecc_subj_sn = X509::subj_sn(env, &key_pair.pub_key)?;
         report_boot_status(LDevIdSubjIdSnGenerationComplete.into());
 
-        let subj_key_id = X509::subj_key_id(env, &key_pair.pub_key)?;
+        let ecc_subj_key_id = X509::subj_key_id(env, &key_pair.pub_key)?;
         report_boot_status(LDevIdSubjKeyIdGenerationComplete.into());
 
         // Generate the output for next layer
         let output = DiceOutput {
-            subj_key_pair: key_pair,
-            subj_sn,
-            subj_key_id,
+            ecc_subj_key_pair: key_pair,
+            ecc_subj_sn,
+            ecc_subj_key_id,
+            mldsa_subj_key_id: [0; 20],
+            mldsa_subj_key_pair: MlDsaKeyPair {
+                key_pair_seed: KEY_ID_LDEVID_MLDSA_KEYPAIR_SEED,
+                pub_key: Default::default(),
```

Comment:
We should actually generate an ML-DSA key and place its public portion here, right?

Reply:
Updated.